lib/landing_session.js

```diff
@@ -317,9 +332,13 @@ export default class LandingSession extends Session {
     const REVIEW_RE = /Reviewed-By\s*:\s*(\S+)/i;
 
     for (const line of metadata) {
-      if (original.includes(line)) {
-        if (line) {
+      if (line.length !== 0 && original.includes(line)) {
+        if (originalHasTrailers) {
           cli.warn(`Found ${line}, skipping..`);
+        } else {
+          cli.error('Git found no trailers in the original commit message, ' +
+                    `but '${line}' is present and should be a trailer.`);
+          process.exit(1);  // make it work with git rebase -x
```

Comment:
It seems reasonable to abort in this case, but if anyone thinks it could be problematic, I'd be happy to revert this particular check.